Program.cs

```diff
@@ -1,4 +1,18 @@
 ﻿using System;
-Console.WriteLine ("Hello, World!");
+using System.Text;
 
-   
\ No newline at end of file
+string ReduceString(string input) {

```

Comment:
Add the function in bottom of main program. We need to add the empty line between the functions.